bin/ssh-wrapper

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Why do we need both ssh-wrapper and ssh-kn?

Reply:
ssh-wrapper could be renamed to "ssh-agent-wrapper", because it is starting and wrapping ssh-agent before running all other commands

Reply:
ssh-agent can not be started in docker-entrypoint since from there we are executing user-commands with gosu (and then the ssh-agent is not being available in subprocess)

Reply:
And if we want to run ansible straight we need the ssh-agent started

Reply:
Renamed wrapper to ssh-agent-wrapper